parkingspaces/src/main/java/com/baspiotr/parkingspaces/domain/services/UserService.java

```diff
@@ -0,0 +1,54 @@
+package com.baspiotr.parkingspaces.domain.services;
+
+import com.baspiotr.parkingspaces.domain.model.Parking;
+import com.baspiotr.parkingspaces.domain.model.User;
+import com.baspiotr.parkingspaces.domain.repository.ParkingRepository;
+import com.baspiotr.parkingspaces.domain.repository.UserRepository;
+import com.baspiotr.parkingspaces.domain.utils.Fee;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class UserService {
+
```

Comment:
ok so now you have services and now lets create Spring Rest Controllers

i.e.

@GetMapping(/fee/{userId})
getFee(@RequestParam(czyjakostak) String userId)
{
 ResponseEntity.ok(userService.getFeeFor(userId)) czy jakos tak
}

Reply:
https://spring.io/guides/gs/rest-service/